### PR TITLE
Add import command to import citizen project skills

### PR DESCRIPTION
**Pull request by NyKo24**

### CODE REVIEW

File: src/Command/ImportCitizenProjectSkillsCommand.php [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    {
+        $output->writeln('Start import...');
+        $countToAdd = 0;
+        foreach (self::CITIZEN_PROJECT_SKILLS as $skillName) {
+            if ($citizenProjectSkill = $this->citizenProjectSkillsRepository->findOneByName($skillName)) {
+                continue;
+            }
+
+            $citizenProjectSkill = new CitizenProjectSkill($skillName);
+            $this->entityManager->getManager()->persist($citizenProjectSkill);
```
Comment (Remg) [inline src/Command/ImportCitizenProjectSkillsCommand.php]: `$this->entityManager->getManager()->persist(new CitizenProjectSkill($skillName));`

File: src/Command/ImportCitizenProjectSkillsCommand.php [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        }
+
+        $this->entityManager->getManager()->flush();
+
+        $output->writeln(sprintf('Finish : %s citizenProjectSkills was added', $countToAdd));
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        $this->entityManager = $this->getContainer()->get('doctrine');
```
Comment (Remg) [inline src/Command/ImportCitizenProjectSkillsCommand.php]: $this->entityManager = $this->getContainer()->get('doctrine.orm.entity_manager');

**Comment by thomasbisignani** [comment]
:ok_hand:

File: src/Command/ImportCitizenProjectSkillsCommand.php [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+        }
+
+        $this->entityManager->flush();
+
+        $output->writeln(sprintf('Finish : %s citizenProjectSkills was added', $countToAdd));
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        $this->entityManager = $this->getContainer()->get('doctrine.orm.entity_manager');
```
Comment (blazarecki) [inline src/Command/ImportCitizenProjectSkillsCommand.php]: Please use `doctrine` instead of `doctrine.orm.entity_manager`

**Pull request merged**